tornado/test/simple_httpclient_test.py

```diff
@@ -704,32 +704,24 @@ def test_port_mapping(self):
         self.assertEqual(response.body, b"Hello world!")
 
 
-class ResolveTimeoutTestCase(AsyncHTTPTestCase):
-    def setUp(self):
-        self.cleanup_event = Event()
-        test = self
+class ResolveTimeoutTestCase(AsyncTestCase):
```

Comment:
this test case doesn't actually need an HTTPServer - although it seems like a duplicate of SimpleHTTPClientTestMixin.test_connect_timeout